enable/gcbench/suite.py

```diff
@@ -52,6 +52,18 @@ def gen_points(count=100):
     return points.reshape(count // 2, 2)
 
 
+def gen_moderate_complexity_path(obj):
+    import math
+
+    obj.arc(150, 100, 50, math.pi, 1.5*math.pi)
+    obj.move_to(150,50)
+    obj.line_to(250, 75)
+    obj.line_to(150, 100)
+    obj.curve_to(115, 75, 135, 125, 100, 100) 
```

Comment:
Nit: flake8 is complaining about this line due to the missing space after the comma on line 59 and the extra space at the end of 62.